File: 2011/03/latex-undoclet/CallGamma.java
import cern.jet.random.Gamma;
import cern.jet.random.engine.MersenneTwister;

public class CallGamma {
  public static void main(String args[]) {
    MersenneTwister mt = new MersenneTwister();
    Gamma gamma = new Gamma(0.5, 0.5, mt);
    System.out.println(gamma.pdf(0));
    System.out.println(gamma.pdf(1));
    System.out.println(gamma.pdf(2));

    System.out.println(gamma.cdf(0));
    System.out.println(gamma.cdf(1));
    System.out.println(gamma.cdf(2));
  }
}

File: 2011/03/latex-undoclet/call-gamma.rb
require 'java'

include_class 'cern.jet.random.Gamma'
include_class 'cern.jet.random.engine.MersenneTwister'

mt = MersenneTwister.new()
gamma = Gamma.new(0.5, 0.5, mt)
gamma.pdf(0)
gamma.pdf(1)
gamma.pdf(2)

gamma.cdf(0)
gamma.cdf(1)
gamma.cdf(2)
